### PR TITLE
Fixes in Python runtime

### LINKED ISSUE #612: Python rounding operators are incorrect
Following #557

The [rounding test](https://github.com/CatalaLang/catala/blob/master/tests/arithmetic/good/rounding.catala_en) fails on the Python backend for now.

### PR DESCRIPTION
**Pull request by rmonat**
Closes #612. 

The type annotations change are a personal preference, feel free to revert.

### CODE REVIEW

**Comment by AltGr** [comment]
Looks fine to me, but @denismerigoux knows the Python backend in more detail than I do

File: runtimes/python/src/catala/runtime.py [hunk 1]
```diff
@@ -78,34 +79,37 @@ def __repr__(self) -> str:
 
 
 class Decimal:
-    def __init__(self, value: Union[str, int, float]) -> None:
-        self.value = mpq(value)
+    def __init__(self, value: Union[str, int, float,Integer]) -> None:
+        if isinstance(value, Integer):
+            self.value = mpq(value.value)
```
Comment (denismerigoux) [inline runtimes/python/src/catala/runtime.py:84]: Good catch

File: runtimes/python/src/catala/runtime.py [hunk 2]
```diff
@@ -416,6 +411,14 @@ def __init__(self, source_position: SourcePosition) -> None:
 # Constructors and conversions
 # ============================
 
+def round(q : Decimal) -> Integer:
```
Comment (denismerigoux) [inline runtimes/python/src/catala/runtime.py:414]: Maybe you could call this function `round_decimal_to_integer`, to not confuse it with `decimal_round`

Comment (rmonat) [reply runtimes/python/src/catala/runtime.py:414]: The name is the same in the ocaml runtime, what shall we do?

Comment (denismerigoux) [reply runtimes/python/src/catala/runtime.py:414]: Oh indeed, I let that slip in a previous PR :) You can leave it as `round` I guess.

**Pull request merged**